config/jobs/kubernetes-csi/csi-lib-utils/csi-lib-utils-config.yaml

```diff
@@ -0,0 +1,17 @@
+presubmits:
+  github.com/kubernetes-csi/csi-lib-utils:
+  - name: pull-sig-storage-csi-lib-utils
+    always_run: true
+    decorate: true
+    skip_report: false
+    spec:
+      containers:
+      # This image was chosen over the more often used kubekins-e2e because
+      # it is smaller and has everything we need (basically just a Go environment).
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20180927-6b4facbe6
```

Comment:
if you just need go, you can also choose to use something like golang:1.11.2

Reply:
I was trying to use something that might have been pulled already, i.e. something which is also used by other jobs. Not sure whether that makes sense?

But I agree, now that all of the special logic is handled by the pod-utils, a generic image should work just as well.

Either way, I'd like to get this merged and perhaps it can be tweaked further later on. We have quite a few repos under kubernetes-csi that might benefit from closer integration with Prow, so I expect that we'll come back to this.

Reply:
You might want to control go versions explicitly - feel free to come back and tune this if you are hitting issues.